src/pages/laced.pug

```diff
@@ -0,0 +1,84 @@
+  .section.section--sm-spacing
+    .container
+      +intro({
+        introImage: {
+          src: "/assets/images/laced/logo.svg",
+          alt: "Laced",
+        },
+        introText: "Laced are determined to bring the sneaker trading experience into the 21st century, and we're helping them to achieve that vision."
+      })
+
+      +details({
+        details: [
+          {
+            detailHeading: "Themes",
+            detailText: "Consultancy, product design, software engineering, service integrations"
+          },
+          {
+            detailHeading: "Deliverables",
+            detailText: "Custom e-commerce platform"
+          },
+          {
+            detailHeading: "Technology",
+            detailText: "React, Ruby on Rails, Stripe, PayPal, UPS"
+          },
+          {
+            detailHeading: "Platform",
+            detailText: "Web"
+          }
+        ]
+      })
+
+  .section
+    .container
+      figure
+        img(src="/assets/images/laced/inline-1.webp" alt="The HEHA! iOS application" onerror="this.onerror=null; this.src='/assets/images/laced/inline-1.jpg'")
+
+      .grid.grid--two-thirds-reverse
+        .grid__item
+          +summary({
+            summaryHeading: "What is it?",
+            summaryText: "Laced allows sneaker enthusiasts to buy and sell collectible sneakers online, as well as track market values, and reserve items at certain prices."
+          })
+
+        .grid__item
+          h4 The challenge
+          p Laced are up against some huge competition from established sneaker trading platforms in the United States. Spotting an opportunity to own the European market, they needed a high quality automated service that could compete with the big players delivered quickly in order to maintain a competitive edge.
+
+          h4 Our approach
+          p Before Laced, European sneaker enthusiasts were having to trade their shoes on the more generalist platforms like eBay, Facebook, or one of a few poorly executed European sneaker trading platforms available.
+
+          p By using a combination of Ruby on Rails and React, we were able to quickly build Laced a high quality online trading platform with a seamless user interface for their customers. Both technologies also allow us to rapidly develop future features and service integrations, setting Laced up with a strong competitive edge.
+
+      figure
+        img(src="/assets/images/laced/inline-2.webp" alt="The HEHA! iOS application" onerror="this.onerror=null; this.src='/assets/images/laced/inline-2.jpg'")
+
+      .grid.grid--two-thirds
+        .grid__item
+          h4 Fully automated payments
+          p As a new name in the industry, loyalty and trust are of extra importance to Laced. To handle payments securely and robustly, we integrated with Stripe and PayPal. Our two integrations allow customers to make immediate purchases, or add credit to their account balance which they can use to reserve items at a certain price.
```

Comment:
The first sentence outlines the importance of trust, but the second doesn't follow on, instead focusing on `secure and robust`. 
Consider:
> We integrated Stripe and PayPal into the platform, allowing customers to purchase with confidence.

Reply:
Without the context of why, the last sentence almost sounds like that was all we could do. I think the whys needs to be spelt out with more clarity:

> Predominantly a credit-based platform, users can also use their Debit or credit card to add funds to their account, allowing them to automatically bid on sneakers within their own pre-defined spend limits. 

^ Wrote that quickly, but I think it explains _why_ they would want to add credit.